app/scripts/models/account.js

```diff
@@ -99,13 +99,6 @@ define([
           });
```

Comment:
Should errors on line 98 still be ignored? Something about that seems fishy. It seems like we should only ignore `UNAUTHORIZED` errors, or other errors we expect instead of blanket ignoring all errors.